service/src/main/java/uk/gov/hmcts/reform/fpl/utils/Constants.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+public class Constants {
```

Comment:
if intention is to keep constants here, then you can change it to interface, then no need for private constructor

Reply:
Good shout